kube-config-files/relations-api.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: relations-api
+  labels:
+    app: relations-api
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: relations-api
+  template:
+    metadata:
+      labels:
+        app: relations-api
+        visualize: "true"
+    spec:
+      containers:
+      - name: relations-api
+        image: coco/relations-api:v1.0.1
+        resources:
+          limits:
+            memory: 256Mi
+        env:
+        - name: APP_PORT
+          value: "8080"
+        - name: NEO_URL
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: neo4j.url
+        - name: GRAPHITE_ADDRESS
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: graphite.address
+        - name: GRAPHITE_PREFIX
+          value: "coco.services.k8s.relations-api"
```

Comment:
We will definitely need to compose this value, by using the environment, as we would probably have multiple environments, and we don't want these metrics to clash